File: backend/src/main/java/edu/cit/olodin/feature/rating/RatingResponse.java
package edu.cit.olodin.feature.rating;

import lombok.AllArgsConstructor;
import lombok.Getter;

import java.time.LocalDateTime;

@Getter
@AllArgsConstructor
public class RatingResponse {
    private Long id;
    private Long listingId;
    private String listingName;
    private Long userId;
    private String userName;
    private Integer rating;
    private String comment;
    private LocalDateTime createdAt;
}

File: NearU_Mobile/app/src/main/java/edu/cit/olodin/nearu/mobile/feature/rating/RatingResponse.kt
package edu.cit.olodin.nearu.mobile.feature.rating

data class RatingResponse(
    val id: Long,
    val listingId: Long,
    val listingName: String,
    val userId: Long,
    val userName: String,
    val rating: Int,
    val comment: String?,
    val createdAt: String?
)